fastlane_bot/helpers/txhelpers.py

```diff
@@ -44,15 +37,13 @@ class TxHelpers:
     __DATE__ = __DATE__
 
     ConfigObj: Config
-    # This is used for the Alchemy SDK
-    network = Network.ETH_MAINNET
 
     def __post_init__(self):
 
         if self.ConfigObj.network.DEFAULT_PROVIDER != "tenderly":
             self.alchemy = Alchemy(
                 api_key=self.ConfigObj.WEB3_ALCHEMY_PROJECT_ID,
-                network=self.network,
+                network=Network.ETH_MAINNET,
```

Comment:
I appreciate that the functionality has not changed, but do we think it is a good idea to hardcode `ETH_MAINNET` in the middle of a generic class?

Reply:
No, this is probably as wrong as it was before.
Can be dealt with in subsequent cleanup tasks (which there will probably several more of).

Reply:
then it probably should be noted down as in issue.